### PR TITLE
modify addresses translations, add new translation for application views

### PR DESCRIPTION
**Pull request by gmichokostas**

### CODE REVIEW

File: config/locales/views/application/de.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 de:
-  address:
-    actions:
+  application:
+    gray_box_layout:
       new: Neue Adresse hinzufügen
```
Comment (gmichokostas) [inline config/locales/views/application/de.yml]: this is strange, I am sure that I deleted the folder first and not just rename it

File: config/locales/views/addresses/de.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+de:
+  addresses:
+    new:
+      heading: Neue Adresse anlegen
+    edit:
+      heading: Adresse bearbeiten
+    actions:
+      new: Neue Adresse hinzufügen
```
Comment (marcaltmann) [inline config/locales/views/addresses/de.yml]: This one can be removed.

File: config/locales/gems/devise/de.yml [hunk 3]
```diff
@@ -29,6 +29,8 @@ de:
         Wir begrüßen Dich herzlich bei Fairmondo, dem fairen Online-Marktplatz in Hand der Nutzer*innen! Bevor Du auf dem Marktplatz handeln kannst, benötigen wir weitere persönliche Daten von Dir, wie z.B. Deine Adresse.
         Diese kannst Du in Deinem Profil eintragen.
     registrations:
+      edit:
+        heading: Deine Versandaddressen
```
Comment (marcaltmann) [inline config/locales/gems/devise/de.yml]: Please change 'Versandaddressen' to 'Versandadressen' (this is a spelling mistake that was in there before). Thanks.

File: config/locales/views/addresses/de.yml (hunk 2 above)
Comment (gmichokostas) [inline config/locales/views/addresses/de.yml]: you mean these two lines: 
```
actions:
      new: Neue Adresse hinzufügen
```

Comment (marcaltmann) [inline config/locales/views/addresses/de.yml]: Yes. Or am I wrong here?

Comment (gmichokostas) [inline config/locales/views/addresses/de.yml]: Yes, you have right, since I added in the other file. Thanks

**Comment by gmichokostas** [comment]
Thanks @marcaltmann!

**Comment by marcaltmann** [comment]
You too

**Pull request merged**